changelog/ctfeWrite.md

```diff
@@ -0,0 +1,24 @@
+# Add `__ctfeWrite` to write output from CTFE
+
+The special function `__ctfeWrite` can now be used to write custom messages to
+`stderr` during CTFE. The function is available in `object.d` and accepts any
```

Comment:
I really think this should be `stdout`, not `stderr`. This is as close to the idea of "output stream" as it can get for this task (compilation). This way you could do `dmd foo.d > someExtraDataToBeUsedLater` and still see any errors.

Reply:
OK, then I suggest not indicating the stream in the documentation.